vendor/vendor.json

```diff
@@ -0,0 +1,268 @@
+{
+	"comment": "",
+	"ignore": "test",
+	"package": [
+		{
+			"checksumSHA1": "E2a5TaCbwqs7PwP1fvfWLhvgr38=",
+			"comment": "v2.2.0-rc.0-72-g106d918",
+			"path": "github.com/coreos/etcd/Godeps/_workspace/src/github.com/coreos/go-systemd/journal",
```

Comment:
"changed the code in Godeps" .. makes me nervous. I wonder what other changes might have been made (to etcd code, etc) that impact etcd-mesos functionality. Did you check for any?